app/models/user.rb

```diff
@@ -1,2 +1,3 @@
 class User < ApplicationRecord
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: I18n.t('is_invalid') }, on: :update
```

Comment:
Have we considered validating the email format whenever there's an email input, rather than only on update? We could potentially remove the on: :update option to ensure consistent email validation across all scenarios where the email might be set or changed. 

something like

`validates :email, format: { with: URI::MailTo::EMAIL_REGEXP, message: I18n.t('is_invalid') }, if: -> { email.present?  }`